src/Libraries/DSIronPython/IronPythonCodeCompletionProviderCore.cs

```diff
@@ -382,7 +382,7 @@ protected override void LogError(string msg)
             Log(msg);
         }
 
-        protected override bool ScopeHasVariable(string name)
+        internal override bool ScopeHasVariable(string name)
```

Comment:
perhaps you want to use `protected internal` - this is an OR access modifier:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/protected-internal